source/index.html.md

```diff
@@ -1,239 +1,674 @@
+
+<aside class="warning">Your Personal Key gives API access to the entire Namara account. If you are thinking about sharing an API Key with another user, consider an Organization API Key or a Project API Key in order to limit their access to your account.</aside>
+
+### Rate Limiting
+
+<!-- check to see if this should go to an email or to the contact page on namara marketing -->
+
+Users are limited to 10,000 requests per month, as well as 100 data set downloads per month. If you exceed this limit, the API will return `status: 429`. If you find yourself meeting the limits, <a href="mailto:sales@namara.io" target="_blank" rel="noreferrer noopener">VERIFY THIS EMAIL</a> to find a solution.
+ 
+# Making Requests
+
+## Parameter Authentication
 
 ```shell
-# With shell, you can just pass the correct header with each request
-curl "api_endpoint_here"
-  -H "Authorization: meowmeowmeow"
-```
+curl -XGET https://api.namara.io/v0/data_sets/:data_set_id?api_key=YOUR_API_KEY
 
-```javascript
-const kittn = require('kittn');
+  or
 
-let api = kittn.authorize('meowmeowmeow');
+curl -i \
+-H "Accept: application/json" \
+-X POST -d "api_key":"YOUR_API_KEY","query":"..." \
+https://api.namara.io/v0/query
 ```
 
-> Make sure to replace `meowmeowmeow` with your API key.
+Append the key as `api_key` to any request body (either `GET` or `POST`) as long as it can be read from the base level params.
+<!-- I'm unclear on what this means and therefore it might warrant explanation -->
+
+## Header Authentication
+
+Append the key to the headers by adding `"X-API-Key":"YOUR_API_KEY"`
+
+## Testing Authentication
+
+Here's how to verify your API Key:
+
+###<div class="colour-pill"><span class="get">GET</span> Verify API Key</div>
+
+Ping the API at `http://api.namara.io/v0/users/verify_token` to ensure that the API token is valid for the user. You don't have to pass `api_key` into the URL - you can pass it in through the headers.
+
+>1)
+`{ "status": true }`
+
+>2)
+`{ "status": false }`
+
+Response | Description
+-------- | -----------
+200: OK | The API Key is valid (*example 1*)
+422: Unprocessable Entity | API does not correspond to any user (*example 2*)
+
+# Data API
+
+> https://:namara_api_host/v0/data_sets/:data_set_id/data/:version
+
+Here, we'll outline the specifications for accessing the Data API. Each data set can be accessed at the API URL. In Namara, click on the "API Info" tab when viewing a data set to see all information regarding ID, version, and properties.
+
+##<div class="colour-pill"><span class="get">GET</span> Data Query</div>
+
+> https://api.namara.io/v0/data_sets/:data_set_id/data/:version
+
+A useful feature is the ability to query a data set by passing in parameters. The query parameters are optional; `data_set_id` and `version` are *path parameters* and **must** be in the call. Here are the ways in which you can manipulate the response:
+
+Parameters | Type | Description
```

Comment:
I would be great to separate these tables into path parameters and query/body parameters

Reply:
done